core/templates/dev/head/App.ts

```diff
@@ -110,6 +134,18 @@ oppia.constant('LOGOUT_URL', '/logout');
 // page visited.
 oppia.constant('ENABLE_PROMO_BAR', true);
 
+// TODO(vojtechjelinek) move these to separate file later
```

Comment:
File issue for this please.

Also make the TODO more explicit -- where to move it to? Is there a pattern to follow? We should establish that.

Reply:
I have created the issue (#6731).